### ISSUE TITLE
Skip link missing id reference

### ISSUE THREAD

**Issue by paul-request** [Legal-and-General/canopy#508]
"Skip link is not keyboard accessible due to the missing id reference to the main content. Without a skip link, it may be difficult for a screen reader or a keyboard user to locate certain points on the page. (AP071, AP072) [My Account Details, Personal Details, Change your Name/Title, Contact Details, Change address – Postcode Lookup, Change address – Manually, Change address – Confirmation & success page, Change address – Specific Home Insurance message, Log in and Security]"

**Comment by paul-request** [comment]
We got this fro man accessibility audit. However, I know that component uses a click event to focus the content. Unsure if this is actually an issue or not?
